.github/pull_request_template.md

```
@@ -0,0 +1,8 @@
+## Brief description
```

Markdownlint

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)